kiva/fonttools/font_manager.py

```diff
@@ -120,6 +120,8 @@ def __init__(self):
 
         self.ttf_lookup_cache = {}
         self.afm_lookup_cache = {}
+        self.ttf_fallback_cache = {}
+        self.afm_fallback_cache = {}
```

Comment:
I should've bought this up earlier but there's no good reason to make these public attributes. Is there?

Reply:
No reason, just following what was already there.